src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -137,7 +151,58 @@ trait RVD {
 
   def crdd: ContextRDD[RVDContext, RegionValue]
 
-  def rdd: RDD[RegionValue]
+  private[rvd] def stabilize(
+    unstable: ContextRDD[RVDContext, RegionValue]
+  ): ContextRDD[RVDContext, RegionValue] = {
+    val localRowType = rowType
+    unstable.map { rv =>
+      val region = Region()
```

Comment:
This needs to be fixed, right?  Who frees this region?

Reply:
Yes.

I think the only way to make this work is to have stabilize send `RegionValue` to `Array[Byte]`. I'll suss out where this creates complications and see if I can resolve them easily.